In [18]:
import pandas as pd
import numpy as np
import seaborn as sns
from scipy import stats
from scipy.stats import norm

# Exercício 1 

In [2]:
dados_salario = np.array([82.1191, 72.8014, 79.1266, 71.3552, 59.192 , 79.1952, 56.518 ,
                          70.3752, 73.5364, 61.0407, 64.3902, 66.4076, 63.5215, 71.9936,
                          60.1489, 78.5932, 76.0459, 67.7726, 64.6149, 80.1948, 76.7998,
                          76.1831, 80.7065, 62.4953, 57.204 , 62.5408, 80.0982, 63.287 ,
                          66.5826, 79.3674])

In [5]:
media = dados_salario.mean()
media

70.14025666666666

In [6]:
dp = np.std(dados_salario)
dp

7.899834598255137

In [12]:
intervalos = norm.interval(0.95, media, stats.sem(dados_salario))
intervalos

(67.26506308375339, 73.01545024957993)

Temos 95% de confiança de que a média salárial das pessoas está no intervalo entre 67.26 e 73.02

In [16]:
amplitude = media - intervalos[0]
amplitude

2.875193582913269